content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -10,187 +10,71 @@ further_reading:
     tag: "Blog"
     text: "Monitor all your CI pipelines with Datadog"
 ---
-{{< callout url="https://app.datadoghq.com/ci/getting-started" >}}
-Intelligent Test Runner for CI Visibility is in private beta. You can request access by completing the form on the CI Visibility Getting Started page.
-{{< /callout >}}
 
-## Overview
+<div class="alert alert-warning">Intelligent Test runner is currently a beta product.</div>
 
-Intelligent Test Runner is Datadog's test impact analysis solution. It allows you to only run the impacted tests for a given commit and skip any that are irrelevant.
+Intelligent Test Runner is Datadog's test impact analysis solution. It automatically selects and runs only the relevant tests needed to validate any given commit, enabling you to reduce testing downtime while maintaining test coverage.
```

Comment:
The text in this page I've basically copied from https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/

Reply:
"Test Impact Analysis" is the industry term for what we're doing, the analysis of which tests are impacted. I think it's helpful to include in terms of SEO (Search engine optimization). Datadog docs pages rank quite high with google, so hopefully when someone searches for "Test impact analysis" our docs page can show up.

I also agree that "testing downtime" is unclear/misleading.

What do you think about this combination of suggestions:

"Intelligent Test Runner is Datadog's test impact analysis solution. It automatically and intelligently selects and runs only the relevant tests for a given commit based on the code being changed. Significantly reduce time spent testing and overall CI costs, while maintaining test coverage."